drivers/sensor/bosch/bmi08x/bmi08x_accel.c

```diff
@@ -536,7 +548,7 @@ static int bmi08x_accel_pm_action(const struct device *dev, enum pm_device_actio
 }
 #endif /* CONFIG_PM_DEVICE */
 
-static DEVICE_API(sensor, bmi08x_api) = {
+static const struct sensor_driver_api bmi08x_api = {
```

Comment:
you must revert this line

Reply:
Commit a553af7 used DEVICE_API to replace defining the api struct